.circleci/config.yml

```diff
@@ -0,0 +1,173 @@
+version: 2
+
+jobs:
+
+  test:
+    docker:
+      - image: rust:latest
+    steps:
+      - checkout
+      # Load cargo target from cache if possible.
+      # Multiple caches are used to increase the chance of a cache hit.
```

Comment:
How does this work / what do you mean here?

Is the second cache a fallback in case there is no cache for the branch?

If that's the case mind just adjusting the comment to clarify that?

Reply:
If you create a new branch, then there won't be a cache for that branch yet. (The branch is in the name of the cache key to prevent two branches overwriting each other's caches.)

However, if you create a new branch, you could still use the latest cache of any other branch as a starting point. You probably won't be able to reuse everything, but - as long as cache invalidation works properly - you'll still be faster than if you don't have any cache at all.

Reply:
I'll add that to the comment.